lib/unorm/unorm.js

```diff
@@ -0,0 +1,894 @@
+(function (root) {
+   "use strict";
+
+/***** unorm.js *****/
+
+/*
+ * UnicodeNormalizer 1.0.0
+ * Copyright (c) 2008 Matsuza
+ * Dual licensed under the MIT (MIT-LICENSE.txt) and GPL (GPL-LICENSE.txt) licenses.
+ * $Date: 2008-06-05 16:44:17 +0200 (Thu, 05 Jun 2008) $
```

Comment:
This one is quite old. Isn't there any more recent lib?

I'm not fan of loading this script everywhere for a specific IE version issue... But on another hand, I'm not sure we can get browser and version on a secure way...

Reply:
Header not updated upstream, this is the last version published yesterday.
Also, your concern about ie only should be now addressed

Reply:
Okay thanks for explanations :)